### DISCUSSION TITLE
Getting the name of a track chunk

### DISCUSSION THREAD

**Discussion by dreammixgames** [melanchall/drywetmidi#162] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/22350412/152659813-2f1df64b-73fc-4fb9-a28c-42b721dcd7d7.png)

Hey. Is there any way to get the **name** of a track chunk, like these above^

**Comment by melanchall** [comment, marked as answer]
Hi,

Just take `SequenceTrackNameEvent` event from a track chunk's events, and then use its `Text` property:

```csharp
var trackName = trackChunk.Events.OfType<SequenceTrackNameEvent>().FirstOrDefault()?.Text;
```

**Comment by dreammixgames** [comment]
Hey Melanchall. I never saw this until now. Just wanted to say thanks!
